### PR TITLE
Move some Config tabs to Twig (part 3)

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

File: templates/pages/setup/general/performance.html.twig
Comment (orthagh) [inline templates/pages/setup/general/performance.html.twig:1]: Every `<th>` in this page should be `<td>` or if you want to keep the semantic tag, at least we should remove the font-weight on sub rows. It's almost unreadable currently

currently: 
image

If i move all th to td:
image

File: templates/pages/setup/general/performance.html.twig [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+{{ fields.largeTitle(__('Performance'), 'ti ti-dashboard') }}
```
Comment (orthagh) [inline templates/pages/setup/general/performance.html.twig:37]:
```suggestion
```

If it's the only main section, it's not required

**Review by orthagh — CHANGES_REQUESTED** [review]

**Review by orthagh — CHANGES_REQUESTED** [review]
Some easy wins for performance tab and ok/failed icons:

Enhance readability:
- declare twig macros to display a failed or icon with a message parameter
- maybe one generic can be named like `icon_message()` and two specifics `success_icon_message` and `warning_icon_message` calling the first.

Enhance UX:
- add  `data-bs-toggle="tooltip"` to have a good render of the message on hover
- add bootstrap `.table-active` class to each tr with a section title

**Review by orthagh — APPROVED** [review]

File: templates/pages/setup/general/performance.html.twig [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+               <td></td>
+               <td></td>
+            </tr>
+         {% endif %}
+      {% endif %}
+      <tr class="table-active">
+         <th colspan="4">{{ __('User data cache') }}</th>
+      </tr>
+      <tr>
+         <th class="fw-normal">{{ __('You can use "php bin/console cache:configure" command to configure cache system.') }}</th>
```
Comment (cedric-anne) [inline templates/pages/setup/general/performance.html.twig:137]: We should prevent translators to be able to alter the command line.
```suggestion
         <th class="fw-normal">{{ __('You can use "%s" command to configure cache system.')|format('php bin/console cache:configure') }}</th>
```

File: templates/pages/setup/general/logs_setup.html.twig [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+   {% endfor %}
+   {% macro log_interval_dropdown(name, label, config, logs_interval_options) %}
+      {{ fields.dropdownArrayField(name, config[name], logs_interval_options, label, {
+         add_field_class: 'purgelog_interval',
+      }) }}
+   {% endmacro %}
+
+   {{ fields.dropdownArrayField('init_all', 0, logs_interval_options, __('Change all'), {
+      label_class: 'col-xxl-5 fst-italic',
+      on_change: 'form_init_all(this.value)'
```
Comment (cedric-anne) [inline templates/pages/setup/general/logs_setup.html.twig:57]: I do not see where `form_init_all` is declared.

File: templates/pages/setup/general/logs_setup.html.twig [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+   }) }}
+
+   {{ fields.smallTitle(__('General')) }}
+   {{ _self.log_interval_dropdown('purge_addrelation', __('Add/update relation between items'), config, logs_interval_options) }}
+   {{ _self.log_interval_dropdown('purge_deleterelation', __('Delete relation between items'), config, logs_interval_options) }}
+   {{ _self.log_interval_dropdown('purge_createitem', __('Add the item'), config, logs_interval_options) }}
+   {{ _self.log_interval_dropdown('purge_deleteitem', __('Delete the item'), config, logs_interval_options) }}
+   {{ _self.log_interval_dropdown('purge_restoreitem', __('Add/update relation between items'), config, logs_interval_options) }}
```
Comment (cedric-anne) [inline templates/pages/setup/general/logs_setup.html.twig:65]:
```suggestion
   {{ _self.log_interval_dropdown('purge_restoreitem', __('Restore the item'), config, logs_interval_options) }}
```

File: templates/pages/setup/general/logs_setup.html.twig [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+   {{ _self.log_interval_dropdown('purge_updateitem', __('Restore the item'), config, logs_interval_options) }}
```
Comment (cedric-anne) [inline templates/pages/setup/general/logs_setup.html.twig:66]:
```suggestion
   {{ _self.log_interval_dropdown('purge_updateitem', __('Update the item'), config, logs_interval_options) }}
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Pull request merged**